src/components/Cart/Steps/Steps.vue

```diff
@@ -16,12 +16,14 @@
 import { useStore } from "@/store";
 import { computed, defineComponent } from "vue";
 import { ComputedRef } from "@vue/reactivity";
+import { useI18n } from "vue-i18n";
 
 export default defineComponent({
   name: "Steps",
 
   setup() {
     const store = useStore();
+    const { tm } = useI18n();
```

Comment:
you use "tm" in one file, just "t" in other one
pick one and stick to it

Reply:
Isn't t and tm something different? I wasn't the one to put it there and I cannot find tm in documentation, so I just want to make sure cuz in case of using Pluralization docs say:
"Your template will need to use $tc() instead of $t()." so maybe tm is similar case.